### PR TITLE
Adding Catalog page in docs

### LINKED ISSUE #7618: [Docs] Add a Catalog page
#### Desired State
A new page should be created on docs describing Meshery Catalog.

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by yash37158**
**Notes for Reviewers**
We are in need of creating a dedicated page for catalogs. Also, it would be great if you can engage yourself with your suggestion to add more content and add steps to create catalogs.
This PR fixes #7618

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 38742959c816acf8b3f853824833e4c05189941f at: https://646e6178cc726b0ff1e26fd0--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 8105cd826a5b762e7ddc1e0e2a1042b072afc26e at: https://646f9a5dd7a93d00757f0165--meshery-docs-preview.netlify.app

**Review by goldfishdolphin — CHANGES_REQUESTED** [review]
@yash37158 thank you for you contribution. @leecalcote and @yash37158  I would make following suggestions :
The title of the page should be Catalog as it is in lowercase .
This [description](https://github.com/layer5io/layer5/pull/4161#issuecomment-1547892457) can be helpful for defining the Catalog page. 
I am not sure if Services Mesh implementations are relevant to this page.
It will be great to add instructions for 
- How to import a catalog using Meshery UI and Meshery CLI
- Add your content to the cloud native catalog

File: docs/pages/concepts/Catalog/catalog.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+## Additional Tools
+
+Apart from service mesh implementations, the catalog also includes a range of tools that complement Meshery and enhance its capabilities:
+
+- [Prometheus](https://prometheus.io/): A popular monitoring and alerting system often used with service meshes.
+- [Grafana](https://grafana.com/): A flexible and powerful visualization tool that works seamlessly with Prometheus.
+
+Browse the Meshery catalog to find more tools and services that can extend your service mesh management capabilities.
+
+If you have any questions or need assistance, please refer to the [Meshery Documentation](/documentation) or reach out to our community for support.
```
Comment (senali-d) [inline docs/pages/concepts/Catalog/catalog.md:45]: @yash37158 Link was break

**Review by senali-d — CHANGES_REQUESTED** [review]

**Comment by senali-d** [comment]
This is a blog drafted #4247 by @leecalcote, "Introducting Meshery Catalog". It might be helpful to further improvement of your page.

**Comment by l5io** [comment]
🚀 Preview for commit bb15b1a6e78957f646ba022b7bca0dbe46c1d621 at: https://6474479dc2068c459081eb1b--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 15792d8971725f977a64e303ff77b758c4c363d1 at: https://64744fedcecdad4ae7d492e5--meshery-docs-preview.netlify.app

File: docs/pages/concepts/Catalog/catalog.md (hunk 1 above)
Comment (yash37158) [reply docs/pages/concepts/Catalog/catalog.md]: yeah i have resolved that in the recent commit

**Comment by yash37158** [comment]
> @yash37158 thank you for you contribution. @leecalcote and @yash37158 I would make following suggestions : The title of the page should be Catalog as it is in lowercase . This [description](https://github.com/layer5io/layer5/pull/4161#issuecomment-1547892457) can be helpful for defining the Catalog page. I am not sure if Services Mesh implementations are relevant to this page. It will be great to add instructions for
> 
> * How to import a catalog using Meshery UI and Meshery CLI
> * Add your content to the cloud native catalog

can you please help me in adding the catalog using meshery UI and CLI

**Comment by yash37158** [comment]
> > @yash37158 thank you for you contribution. @leecalcote and @yash37158 I would make following suggestions : The title of the page should be Catalog as it is in lowercase . This [description](https://github.com/layer5io/layer5/pull/4161#issuecomment-1547892457) can be helpful for defining the Catalog page. I am not sure if Services Mesh implementations are relevant to this page. It will be great to add instructions for
> > 
> > * How to import a catalog using Meshery UI and Meshery CLI
> > * Add your content to the cloud native catalog
> 
> can you please help me in adding instructions for the catalog using meshery UI and CLI

**Comment by goldfishdolphin** [comment]
> > > @yash37158 thank you for you contribution. @leecalcote and @yash37158 I would make following suggestions : The title of the page should be Catalog as it is in lowercase . This [description](https://github.com/layer5io/layer5/pull/4161#issuecomment-1547892457) can be helpful for defining the Catalog page. I am not sure if Services Mesh implementations are relevant to this page. It will be great to add instructions for
> > > 
> > > * How to import a catalog using Meshery UI and Meshery CLI
> > > * Add your content to the cloud native catalog
> > 
> > 
> > can you please help me in adding instructions for the catalog using meshery UI and CLI

You can find instructions if you click at the link 'import any catalog' item at https://meshery.io/catalog.

**Comment by l5io** [comment]
🚀 Preview for commit b26c404dffa7c8af112dcef08055ab5f7f47e97c at: https://6474ff867d4ce04744319f9f--meshery-docs-preview.netlify.app

**Comment by yash37158** [comment]
Hey @senali-d @leecalcote @goldfishdolphin I have added instructions for Both UI and CLI let me know if we have any more fix

**Review by senali-d — COMMENTED** [review]
Great job @yash37158. 

Suggestion from my side. It's better if you can provide the link to catalog page in [meshery.io](https://meshery.io) ([calalog](https://meshery.io/catalog)) here.

Let's get others feedback also.

**Comment by goldfishdolphin** [comment]
Well done @yash37158 .

**Comment by l5io** [comment]
🚀 Preview for commit b279da7cbae2e121316475cbfc46a2f41b3be89c at: https://64750d312837564dcaa30d7e--meshery-docs-preview.netlify.app

**Comment by yash37158** [comment]
> Well done @yash37158 .

Thanks  🚀

**Comment by yash37158** [comment]
Are we good to go ?

**Comment by l5io** [comment]
🚀 Preview for commit f90adee3dbce9aea646a761084615da9cec5c747 at: https://647647b239185007bc0e5af3--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 3260454d6f3f3f7fcfb8efffc17c1a674aead355 at: https://6476f236143d22703f5716df--meshery-docs-preview.netlify.app

**Comment by yash37158** [comment]
// @leecalcote

**Comment by yash37158** [comment]
Hey @senali-d @leecalcote @goldfishdolphin It's been a week, let me know if we need more addons

**Comment by Chadha93** [comment]
@yash37158 Let's discuss the update of this PR on the Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

**Comment by senali-d** [comment]
@leecalcote  has mention important points regarding the catalog doc in meshery development meeting. I think you might make sure those areas are covered in your pr. // @yash37158 

Here is the link to [Meshery Development Meeting](https://www.youtube.com/watch?v=hYBACqrxgMI)

File: docs/pages/concepts/Catalog/catalog.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: Catalog
+permalink: concepts/catalog
+redirect_from: catalog
+type: concepts
+abstract: Getting to know about catalog
```
Comment (leecalcote) [inline docs/pages/concepts/Catalog/catalog.md:7]:
```suggestion
abstract: Browsing and using cloud native patterns
```

File: docs/pages/concepts/Catalog/catalog.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+1. Ensure that you have Meshery CLI installed on your machine and it is configured to connect to your desired Meshery instance.
+2. Open a terminal or command prompt.
+3. Use the Meshery CLI commands to interact with the catalog. `mesheryctl pattern`
+4. Follow the prompts or instructions provided by the Meshery CLI help.
+* Apply pattern file:  `mesheryctl pattern apply --file [path to pattern file | URL of the file]`
+* Delete pattern file:  `mesheryctl pattern delete --file [path to pattern file]`
+* View pattern file:  `mesheryctl pattern view [pattern name | ID]`
+* List all patterns: `mesheryctl pattern list`
+5. Onboarding an application. `mesheryctl app onboard -f [file-path]`
+6. Applying WASM Filter. `mesheryctl exp filter apply --file [GitHub Link]`
```
Comment (leecalcote) [inline docs/pages/concepts/Catalog/catalog.md:53]: Please make these local links to other pages in the docs.

File: docs/pages/concepts/Catalog/catalog.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+It also supports a collaborative environment, where DevOps engineers can share their experiences, feedback, and best practices with others in the community. Import cloud native patterns published by others into your Meshery Server. Benefit from and build upon each pattern by incorporating your own tips and tricks, then publish and share with the community at-large. This facilitates knowledge-sharing and helps to build a strong ecosystem of cloud native infrastructure experts.
+
+## Add your content to the catalog
+
+To add your content to the Cloud Native Catalog -
+
+1. Head over [meshery](https://github.com/meshery/meshery.io/blob/master/README.md#add-your-content-to-the-cloud-native-catalog) repository and set up your local enviroment.
+1. Create a new file in the catalog collection, under the respective type.
+2. Follow the template in the [_defaults.md](https://github.com/meshery/meshery.io/blob/master/collections/_catalog/_defaults.md) and fill out the details.
+3. Add your pattern file to the [catalog](https://github.com/meshery/meshery.io/tree/master/catalog) folder so that others can download the file.
```
Comment (leecalcote) [inline docs/pages/concepts/Catalog/catalog.md:29]: This is no longer the process. Please see the instructions on meshery.io/catalog, @yash37158

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: docs/pages/concepts/Catalog/catalog.md (hunk 4 above)
Comment (yash37158) [reply docs/pages/concepts/Catalog/catalog.md:29]: Thanks for your response @lee, i definitely take that into consideration

**Comment by yash37158** [comment]
Sure @senali-d sorry for the delay, just caught with final exams, i wrap this asap

**Comment by vishalvivekm** [comment]
> Sure @senali-d sorry for the delay, just caught with final exams, i wrap this asap

updates? @yash37158

**Review by samyakshah3008 — APPROVED** [review]
Great work! 💪 
Looks good to merge ✨

**Comment by l5io** [comment]
🚀 Preview for commit 510989f3a22f5ac5406e977ad3835a2756b2ab98 at: https://6486e451561c6a0a3ba5ee41--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 510989f3a22f5ac5406e977ad3835a2756b2ab98 at: https://6486e462c8c20b0532f1c024--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit ad4ca4b9fbb48ccdad6083556540b44401fe4072 at: https://6486f00158c5bd1296c7f050--meshery-docs-preview.netlify.app

**Comment by vishalvivekm** [comment]
@yash37158 please be there on the call: we'll discuss about the catalog page

**Review by goldfishdolphin — APPROVED** [review]
Looking good :)

**Comment by yash37158** [comment]
Sure @vishalvivekm

**Comment by l5io** [comment]
🚀 Preview for commit 9d1fd7ed5d34f304f731e5e2b258a7cfef0b0335 at: https://648704f9087f7e21ea1ffe9f--meshery-docs-preview.netlify.app

**Comment by thisiskaransgit** [comment]
@yash37158 have you incorporated all the requests?

**Pull request merged**